Scripts/flowlauncher.nuspec

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <metadata>
-    <id>FlowLauncher</id>
+    <id>Flow-Launcher</id>
```

Comment:
This is causing the install folder to be 'Flow' instead of 'FlowLauncher', using '-' is not supported by nuget

Reply:
Good catch. Should the plugin be `FlowLauncher.Plugin` for consistency?

Reply:
I think keeping it Flow.Launcher.Plugin is better. For me I chose FlowLauncher for ID because Flow.Launcher as a folder name in Local seems a bit weird/improper, obviously Flow-Launcher or Flow Launcher is more ideal but unfortunately not supported.